data/json/mapgen/lab/lab_floorplans.json

```diff
@@ -51,7 +51,7 @@
     }
   },
   {
-    "//": "military bunker with 1-in-10 chance of the doors being smashed in by a hulk.",
```

Comment:
Why did you remove the extended comments? They are comments, after all.

Reply:
This change was not me, If you check CleverRaven's master you will see the change was made recently (I believe when metal doors were switched out for frosted glass doors). The reason you see the change here is because my lab_floorplans.json was not updated the first commit but I recommited my changes with the new version of this file.